src/app/pages/MediaArticlePage/MediaArticlePage.tsx

```diff
@@ -161,7 +161,7 @@ const MediaArticlePage = ({ pageData }: { pageData: Article }) => {
 
   const bylineLinkedData = bylineExtractor(bylineContribBlocks);
 
-  const hasByline = !!bylineLinkedData;
+  const hasByline = !!bylineLinkedData[0];
```

Comment:
Should we check the length of the byline linked data instead?

Reply:
Have refactored the bylineExtractor to just return null now instead of [null] , so this change isnt needed at all anymore 🎉